instrumentation/runtime-metrics/javaagent/src/main/java/io/opentelemetry/instrumentation/javaagent/runtimemetrics/RuntimeMetricsInstaller.java

```diff
@@ -33,6 +34,7 @@ public void afterAgent(Config config, AutoConfiguredOpenTelemetrySdk unused) {
       Classes.registerObservers(GlobalOpenTelemetry.get());
       Cpu.registerObservers(GlobalOpenTelemetry.get());
       MemoryPools.registerObservers(GlobalOpenTelemetry.get());
+      BufferPools.registerObservers(GlobalOpenTelemetry.get());
```

Comment:
We currently enable metrics by default only when opentelemetry specification defines semantic conventions for them https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/runtime-environment-metrics.md Move registering these down to where `GarbageCollector.registerObservers` is called

Reply:
Just want to make sure my understanding is correct, do you mean all the four needs to be moved to where `GarbageCollector.registerObservers` is called? As I can see all those metrics are in `experimental` status